chain-signatures/node/src/protocol/signature.rs

```diff
@@ -703,7 +703,7 @@ impl SignatureManager {
         // add back the failed presignatures that were incompatible to be made into
         // signatures due to failures or lack of participants.
         for presignature in failed_presigs {
-            presignature_manager.insert_mine(presignature).await;
+            presignature_manager.insert(presignature, true).await;
```

Comment:
This interface was separated on porpouse. `insert_mine(presignature)` is more readable then `insert(presignature, true)`.

Reply:
I personally like insert(x, bool), since if so we don't need to remember to do insert_mine() rather than insert() when the presig or triple is mine.